### PR TITLE
Allow different wps server dialects

### PR DESCRIPTION
**Pull request by nbrinckm**
This MR can allow to handle different parts of the wps execute requests differently depending if it is based on the gfz wps server (based on the 52N wps 4.0) or the javaps.

### CODE REVIEW

File: asyncwrapper/src/main/kotlin/org/n/riesgos/asyncwrapper/config/WPSConfiguration.kt [hunk 1]
```diff
@@ -5,4 +5,4 @@ import org.springframework.boot.context.properties.ConstructorBinding
 
 @ConstructorBinding
 @ConfigurationProperties(prefix = "wps")
-data class WPSConfiguration(val wpsURL: String, val process: String, val wpsVersion : String, val retryConfiguration: RetryConfiguration)
+data class WPSConfiguration(val wpsURL: String, val process: String, val wpsVersion : String, val dialect: String, val retryConfiguration: RetryConfiguration)
```
Comment (arnevogt) [inline asyncwrapper/src/main/kotlin/org/n/riesgos/asyncwrapper/config/WPSConfiguration.kt:8]: maybe use an enum for dialects

**Review by arnevogt — COMMENTED** [review]
I think it might be better to introduce a enumeration for WPS dialects.
Though, I think you can merge this in order to resolve the issue.

(Did not see your comment on the issue)

**Comment by arnevogt** [comment]
@MichaelLangbein 
If solves the issues this can be merged. 
Maybe we can clean up the code a bit (see the comments above). However, we can also open a new PR for this.

**Comment by MichaelLangbein** [comment]
> @MichaelLangbein If solves the issues this can be merged. Maybe we can clean up the code a bit (see the comments above). However, we can also open a new PR for this.

Yeah, this branch does indeed solve issue #66 . Merging and creating a new issue for the remaining [comment](https://github.com/riesgos/async/pull/71#pullrequestreview-1334376423).

**Pull request merged**